### PR TITLE
[FIX] Fix loading loop in `SllmLocalStore`

### PR DESCRIPTION
**Pull request by future-xy**
## Description
This PR enhances the `SllmLocalStore` by addressing issues in the loading loop and adapting to the latest automatic hardware configuration detection feature.

## Motivation
- Fixes the first issue mentioned in #131.
- Prepares the codebase for the upcoming live-migration feature release.

## Type of Change
- [x] Bug fix
- [x] Enhancement

## Checklist
- [x] I have read the [CONTRIBUTING](https://github.com/ServerlessLLM/ServerlessLLM/blob/main/CONTRIBUTING.md) guide.
- [x] The changes are tested and verified to maintain compatibility with existing features.

### CODE REVIEW

File: sllm/serve/store_manager.py [hunk 1]
```diff
@@ -40,37 +40,41 @@ def __init__(
...
+
     async def register_model(
         self, model_name: str, backend: str, backend_config
     ):
         async with self.lock:
             if model_name in self.disk_models:
                 logger.error(f"{model_name} already registered")
                 return
-            model_path = os.path.join(backend, model_name)
+            model_path = await self._get_model_path(model_name, backend)
```
[the remark this answers is not in the document]
Comment (future-xy) [reply sllm/serve/store_manager.py:77]: Good point. I've modified this.

File: sllm/serve/hardware_info_collector.py
[the remark this answers is not in the document]
Comment (future-xy) [reply sllm/serve/hardware_info_collector.py:1]: deleted

**Review by JustinTong0323 — APPROVED** [review]
LGTM.

File: sllm/serve/hardware_info_collector.py [hunk 2]
```diff
@@ -21,9 +21,8 @@ def collect_all_info():
     Collect all hardware information and return as a dictionary.
     """
     hardware_info = {}
-    hardware_info["host_size"] = get_memory_info()
-    hardware_info["host_bandwidth"] = benchmark_memory_bandwidth()
-    hardware_info["pcie_bandwidth"] = "N/A"  # TODO: Not implemented
+    # TODO: collect pcie_bandwidth
+    hardware_info["pcie_bandwidth"] = 25000000000  # 25 GB/s, PCIe 4.0 x16
```
Comment (SecretSettler) [inline sllm/serve/hardware_info_collector.py:25]: Can it be benchmarked? Can we do simple tensor h2d tests using pytorch?

**Pull request merged**